frontend_vue/src/main.ts

```diff
@@ -7,12 +7,18 @@ import registertBaseComponents from '@/components/base/_index.js';
 import { library } from '@fortawesome/fontawesome-svg-core'
 import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
 import { faBars, faXmark, faLink } from '@fortawesome/free-solid-svg-icons';
+import { createVfm } from 'vue-final-modal'
```

Comment:
what is this?

Reply:
from the final modal doc
`In vue-final-modal 4, it's necessary to create a plugin by createVfm() and register it because vue-final-modal 4 has some shared context that is based on provide/inject.`